README.md

```diff
@@ -1,138 +1,49 @@
 # AutoGVP: Automated Germline Variant Pathogenicity
-Jung Kim, Ammar S. Naqvi, Ryan J. Corbett, Rebecca Kaufman, Zalman Vaksman, Miguel A. Brown,  Daniel P. Miller, Zhuangzhuang Geng, Phillip B. Storm, Adam C. Resnick, Jo Lynne Rokita, Douglas R. Stewart, Sharon J. Diskin
+Jung Kim, Ammar S. Naqvi, Ryan J. Corbett, Rebecca Kaufman, Zalman Vaksman, Miguel A. Brown,  Daniel P. Miller, Zhuangzhuang Geng, Phillip B. Storm, Adam C. Resnick, Douglas R. Stewart, Jo Lynne Rokita, Sharon J. Diskin
 
 ## AutoGVP Workflow  
 <img src="https://github.com/diskin-lab-chop/pathogenicity-assessment/blob/6848a369885fb9e03e75ec82afc97379457c0275/figures/germline-pathogenecity_flow.png" align="center" width = "600">
 
+For more detailed instructions, please visit the [user guide](https://github.com/diskin-lab-chop/AutoGVP/wiki/User-Guide) on our wiki.
+
 ## Clone the AutoGVP repository
 ```bash
 git clone https://github.com/diskin-lab-chop/AutoGVP.git
 ```
 
 ## Docker set-up
-
-### docker pull
+### docker pull, run, and execute
 ```bash
 docker pull pgc-images.sbgenomics.com/naqvia/autogvp:latest
 ```
 cd to your clone of `AutoGVP`
```

Comment:
would change to `### cd to your clone of `AutoGVP``

Reply:
weird, this should be for line #19.

Reply:
the cd happens within the running so I didn't make it a header